mls_cloud_folder: /data/digiforest/steim_am_rhein/2023-08-08-16-14-48-exp03/payload_clouds_in_map
offset: [0,0,0]
pose_graph_file: /data/digiforest/steim_am_rhein/2023-08-08-16-14-48-exp03/transformed_clouds_payloads_in_map/pose_graph.g2o
output_folder: /data/digiforest/steim_am_rhein/2023-08-08-16-14-48-exp03/optimized_payloads_in_map
load_clouds: True
tiles: False
